wandb/data_types.py

```diff
@@ -1002,30 +1006,28 @@ def seq_to_json(cls, images, run, key, step):
             num_images_to_log = Image.MAX_THUMBNAILS
             images = images[:num_images_to_log]
 
-        if width * num_images_to_log > Image.MAX_DIMENSION:
-            max_images_by_dimension = Image.MAX_DIMENSION // width
-            logging.warning('Only {} images will be uploaded. The maximum total width for a set of thumbnails is 65,500px, or {} images, each with a width of {} pixels.'.format(
-                max_images_by_dimension, max_images_by_dimension, width))
-            num_images_to_log = max_images_by_dimension
-
-        total_width = width * num_images_to_log
-        sprite = PILImage.new(
-            mode='RGB',
-            size=(total_width, height),
-            color=(0, 0, 0))
-        for i, image in enumerate(images[:num_images_to_log]):
-            location = width * i
-            sprite.paste(image._image, (location, 0))
-        fname = '{}_{}.png'.format(key, step)
-        # fname may contain a slash so we create the directory
-        util.mkdir_exists_ok(os.path.dirname(os.path.join(base, fname)))
-        sprite.save(os.path.join(base, fname), transparency=None)
-        meta = {"width": width, "height": height, "format": "png",
-                "count": num_images_to_log, "_type": "images"}
-        # TODO: hacky way to enable image grouping for now
-        grouping = images[0]._grouping
-        if grouping:
-            meta["grouping"] = grouping
+        for i, obj in enumerate(images):
+            if not obj.is_bound():
+                obj.bind_to_run(run, key, step, id_=i)
+
+        jsons = [obj.to_json(run) for obj in images]
+
+        for obj in jsons:
+            if not obj['path'].startswith(cls.get_media_subdir()):
```

Comment:
This change is in master and is required for this to work in windows:

https://github.com/wandb/client-ng/blob/master/wandb/data_types.py#L530